ui/raidboss/data/07-dt/trial/queen-eternal-ex.ts

```diff
@@ -784,13 +784,26 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
         'Ice Pillar': 'pilier de glace',
         'Queen Eternal': 'Reine Éternité',
         'Virtual Boulder': 'roche instable',
       },
       'replaceText': {
+        '\\(Dorito Stack\\)': '(Package donut)',
+        '\\(Flares/Stack\\)': '(Brasiers/Packages)',
+        '\\(Knockback\\)': '(Poussée)',
+        '\\(big\\)': '(gros)',
+        '\\(cast\\)': '(Incante)',
+        '\\(damage\\)': '(Dommage)',
+        '\\(front\\)': '(Devant)',
+        '\\(left tower\\)': '(Tour gauche)',
+        '\\(motion\\)': '(Déplacement)',
+        '\\(orb\\)': '(Orbe)',
+        '\\(platforms\\)': '(Platformes)',
+        '\\(right tower\\)': '(Tour gauche)',
```

Comment:
The `left tower` and the `right tower` have the same translation, please have a look.

Reply:
Awwww, good catch. Thanks 👍